addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -276,6 +288,12 @@ img.ms-auto, img.mx-auto {
 .base-fs {
     @include font-size($font-size-base);
 }
+// Equivalent "font-size" only for the Odoo own "o_small" class. Note that the
+// "o_small" class currently also changes the font-size only but this is the
+// stay consistent with the other classes which act that way (as display-x).
+.o_small-fs {
```

Comment:
Don't like the mix of `-` and `_` either... but seems consistent with the rest :shrug:  It's also a class that is not to be used in Odoo layouts anyway (just used by the user using the builder).